src/components/Poster.js

```diff
@@ -1,9 +1,15 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
 import './Poster.css';
 
-const Poster = ({ image, id, title }) => {
+const Poster = ({ movie }) => {
   return (
-      <img className='poster' src={image} alt={title}/>
+    <Link to={{
+        pathname:`/movie-details/${movie.title.replace(/\s+/g, '')}`,
+        state: {movie}
+      }}>
+      <img src={movie.poster_path} className='poster'/>
+    </Link>
```

Comment:
Congrats on figuring out router and how to make an image an Link with router also!